src/components/ContributeAs.js

```diff
@@ -0,0 +1,249 @@
+import PropTypes from 'prop-types';
```

Comment:
Missing `React` import

Reply:
It's not required for Next.js or Styleguidist.

Reply:
The goal of styled components being for them to be reusable, I think we should enforce importing React in them. We will have problems if we start using things like https://bitsrc.io to share our components.

Whatever we choose, eslint `react/react-in-jsx-scope` rule is currently generating a warning for this. So we should either import React or disable this rule.

Reply:
The `react/react-in-jsx-scope` rule is active on purpose and we should then add the import.

At this stage, all `eslint` warnings are carefully considered, they should all be fixed when contributing new code and refactoring old one.